src/primitive_crypto.cc

```diff
@@ -121,31 +160,53 @@ PRIMITIVE(siphash_get) {
   return result;
 }
 
-AesCbcContext::AesCbcContext(
+AesEcbContext::AesEcbContext(
     SimpleResourceGroup* group,
-    const uint8* key,
-    const uint8* iv,
+    const Blob* key,
     bool encrypt)
   : SimpleResource(group) {
```

Comment:
Not your code, but I think I'd merge this with the previous line.

Reply:
Done.